<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>608</height>
     </rect>
    </property>
    <widget class="QPushButton" name="autorisatedPushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>280</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Авторизация</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fon">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>441</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="SettingsPushButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>410</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Настройки</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>64</y>
       <width>451</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>27</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.601, y1:0, x2:0.495, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(72, 117, 0);
color: rgb(67, 168, 0);
color: rgb(2, 253, 224);
color: rgb(0, 49, 212);
background-color: rgb(1, 200, 225);
</string>
     </property>
     <property name="text">
      <string>Concurrent Betta  V1.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="selectionKSApushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Создать модель</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>361</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top-color: rgb(0, 85, 255);
color: rgb(0, 85, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="exitPushButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>470</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>выйти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openKSApushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>360</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Открыть модель</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>474</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Menu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autorisatedPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Menu</receiver>
   <slot>autorisate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectionKSApushButton</sender>
   <signal>clicked()</signal>
   <receiver>Menu</receiver>
   <slot>OpenBuidKSA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>autorisate()</slot>
  <slot>OpenBuidKSA()</slot>
 </slots>
</ui>
